agent/consul/state/catalog.go

```diff
@@ -841,19 +845,30 @@ func (s *Store) ServicesByNodeMeta(ws memdb.WatchSet, filters map[string]string)
 }
 
 // maxIndexForService return the maximum Raft Index for a service
-// If the index is not set for the service, it will return:
-// - maxIndex(nodes, services) if checks is false
-// - maxIndex(nodes, services, checks) if checks is true
-func maxIndexForService(tx *memdb.Txn, serviceName string, checks bool) uint64 {
-	transaction, err := tx.First("index", "id", serviceIndexName(serviceName))
-	if err == nil {
-		if idx, ok := transaction.(*IndexEntry); ok {
-			return idx.Value
-		}
+// If the index is not set for the service, it will return the missing
+// service index.
+// The service_last_extinction is set to the last raf index when a service
```

Comment:
```suggestion
// The service_last_extinction is set to the last raft index when a service
```

Reply:
Once this typo is fixed I will merge this.